docs/dApps/unity/quickstart.md

```diff
@@ -0,0 +1,373 @@
+       .Wallet
+       .GetActiveAddress();
+
+   const string imageAddress = "ipfs://QmX4t8ikQgjvLdqTtL51v6iVun9tNE7y7Txiw4piGQVNgK";
+
+   var tokenMetadata = new TokenMetadata
+   {
+       Name = "My token",
+       Description = "Description for my token",
+       Symbol = "MYTOKEN",
+       Decimals = "0",
+       DisplayUri = imageAddress,
+       ArtifactUri = imageAddress,
+       ThumbnailUri = imageAddress
+   };
+
+   TezosManager
+       .Instance
+       .Tezos
+       .TokenContract
+       .Mint(
+           completedCallback: OnTokenMinted,
+           tokenMetadata: tokenMetadata,
+           destination: initialOwner,
+           amount: 100);
+
+   private void OnTokenMinted(TokenBalance tokenBalance)
+   {
+       Debug.Log($"Successfully minted token with Token ID {tokenBalance.TokenId}");
+   }
+   ```
+
+   For a complete example of creating tokens, see the file `TezosSDK/Examples/Contract/Scripts/MintToken.cs` and the Contract example scene.
+
+   To store files such as images or metadata that are associated with the token, you can set up distributed storage with the InterPlanetary File System (IPFS).
+   For example, [Pinata](https://www.pinata.cloud/) provides an API to store files in IPFS.
+   The tutorial [Create a contract and web app that mints NFTs](../../tutorials/create-an-nft/nft-taquito) covers setting up an IPFS account and storing files on IPFS.
+
+   For example, this code uses the `TezosSDK.Scripts.FileUploaders.UploaderFactory` class and Pinata to open a window that allows the user to select a file to upload to IPFS.
+
+   ```csharp
+   var uploader = UploaderFactory.GetPinataUploader("Your pinata api key");
+
+   uploader.UploadFile(imageAddress =>
+   {
+      Debug.Log($"File uploaded to ipfs, address is {imageAddress}");
+   });
+   ```
+
+   You can also use other tools such as the Pinata API to upload files to IPFS directly.
+
+1. To transfer tokens, call the contract's `Transfer` entrypoint.
+
+   Like the `mint` entrypoint, the built-in contract has a `transfer` method that accepts these parameters and calls the deployed contract's `transfer` entrypoint:
+
+      - A callback function to run when the transfer is complete
+      - The account to transfer the tokens to
+      - The ID of the token
+      - The amount of tokens to transfer
+
+   This example transfers 12 tokens with the ID 5 to the account in the variable `destinationAccountAddress`.
+
+   ```csharp
+   public void HandleTransfer()
+   {
+       TezosManager
+           .Instance
+           .Tezos
+           .TokenContract
+           .Transfer(
+               completedCallback: TransferCompleted,
+               destination: destinationAccountAddress,
+               tokenId: 5,
+               amount: 12);
+   }
+
+   private void TransferCompleted(string txHash)
+   {
+       Logger.LogDebug($"Transfer complete with transaction hash {txHash}");
+   }
+   ```
+
+   For a complete example, see the Transfer example scene.
```

Comment:
This should be a link to the page.

Reply:
I'll link it when we have a separate page for the scenes.